weave-js/src/components/PagePanelComponents/Home/Browse3/pages/wfReactInterface/tsDataModelHooks.ts

```diff
@@ -669,6 +670,13 @@ const useOpVersion = (
       };
     }
 
+    if (opVersionRes.obj == null) {
```

Comment:
This block (and the one below that is similar basically handles the case when the target data is not found. before, this would create a JS error due to incorrect nulls

Reply:
moved to new PR